docs/Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 1.12.0 (Not released yet)
 -------------------------
+* [#4540](https://github.com/TouK/nussknacker/pull/4540) Improvement: Allow to select a claim from OIDC JWT to represent the username
```

Comment:
Improvement: Allow selecting a claim from OIDC JWT to represent the username

Reply:
done